view/template/mail/_settingsForm.php

```diff
@@ -0,0 +1,12 @@
+<?php $error = $error ?? null ?>
+<?php $tag = $tag ?? null ?>
+<?php $largeInput = $largeInput ?? false ?>
+<form id="settings_form" action="/list/subscribe" method="POST" novalidate>
+  <?php if ($error): ?>
+    <div class="notice notice-error spacer1"><?php echo $error ?></div>
+  <?php endif ?>
+  <div class="mail-submit" >
+    <input name="receive" type="radio" value="true" checked><label>Receive Email</label><br>
```

Comment:
Should probably just be a single submit button, though I suppose there could be a cancel button too.